app/src/components/welcome-page/welcome-page.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from '../button';
+import Copyright from '../copyright';
+
+export default function WelcomePage(): JSX.Element {
+    return (
+        <div className="ui">
+            <div className="ui__inner">
+                <div className="ui__top" />
+                <div className="ui__center">
+                    <div className="welcome--page__title t-title">
+                        Castle
+                        <br />
+                        of
+                        <br />
+                        Ghosts
+                    </div>
+                    <div className="m--t__5" />
+                    <Button>Start</Button>
+                    <div className="m--t__5" />
+                    <div className="link">Leaderboard</div>
+                </div>
+                <div className="ui__bottom">
```

Comment:
```suggestion
                <footer className="ui__bottom">
```

В целом полностью футер можно вынести в отдельный компонент

Reply:
А он у нас больше нигде в таком виде не встречается. Может больно жирно ему свой компонент? :)

Reply:
Футер можно на каждой странице прилепить. А тут возможно все же не совсем футер, я не нашел класса ui__bottom